File: sdk/javascript-closure-angular/API/Client/Order.js
goog.provide('API.Client.Order');

/**
 * @record
 */
API.Client.Order = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.Order.prototype.ID;

/**
 * @type {!string}
 * @export
 */
API.Client.Order.prototype.type;

/**
 * @type {!string}
 * @export
 */
API.Client.Order.prototype.fromUserID;

/**
 * @type {!string}
 * @export
 */
API.Client.Order.prototype.billingAddressID;

/**
 * @type {!string}
 * @export
 */
API.Client.Order.prototype.shippingAddressID;

/**
 * @type {!string}
 * @export
 */
API.Client.Order.prototype.comments;

/**
 * @type {!number}
 * @export
 */
API.Client.Order.prototype.shippingCost;

/**
 * @type {!number}
 * @export
 */
API.Client.Order.prototype.taxCost;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.Order.prototype.xp;

File: sdk/async-scala/src/main/scala/io/swagger/client/model/Order.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Order (
  ID: Option[String],
Type: Option[String],
FromUserID: Option[String],
BillingAddressID: Option[String],
ShippingAddressID: Option[String],
Comments: Option[String],
ShippingCost: Option[Number],
TaxCost: Option[Number],
xp: Option[Any]
)
